src/model/manager.model.ts

```diff
@@ -10,6 +10,20 @@ export interface CreateBikeBody {
   location: string;
 }
 
+export interface RegisterUserBody {
+  userName: string;
+  email: string;
+  password: string;
+  type?: UserType;
+}
+
+export const RegisterUserBodySchema = Joi.object({
```

Comment:
Make their name like ManagerRegisterUserBody so that we can differentitate that it is form manager side